src/js/Core/Storage/SyncedStorage.js

```diff
@@ -284,6 +284,9 @@ SyncedStorage.defaults = Object.freeze({
     "profile_showcase_own_twitch": false,
     "profile_showcase_twitch_profileonly": false,
 
+    "profile_accurate_status": true,
+    "profile_shared_game": true,
```

Comment:
```suggestion
    "profile_shared_game": false,
```

Not worth the extra request in most cases